demo/sticky.html

```diff
@@ -54,12 +68,16 @@
 
   <div>
     <button id=swapButton>switch virtual-scroller &lt;-&gt; div</button>
+    <button id=perfDemoModeButton>performance demo mode</button>
   </div>
 
-  <virtual-scroller id=scroller></virtual-scroller>
-
+  <fps-display style="display: none"></fps-display>
```

Comment:
Personally I would use the `hidden` attribute (plus `toggleAttribute("hidden", perfModeDemoState)`)

Reply:
I started to do that but then that either gives me 2 indepdent bits for the state of whether we're in demo mode or not (one in JS, one in this attribute) or I can switch to make this the canonical bit but that feels weird.

But I did switch to toggling the hidden attribute.